bot/exts/evergreen/emoji.py

```diff
@@ -1,31 +1,33 @@
-import datetime
 import logging
 import random
+import textwrap
 from collections import defaultdict
-from typing import List, Tuple
+from datetime import datetime
+from typing import List, Optional, Tuple
 
-import discord
+from discord import Color, Embed, Emoji
 from discord.ext import commands
 
 from bot.constants import Colours, ERROR_REPLIES
 from bot.utils.pagination import LinePaginator
+from bot.utils.time import time_since
 
 log = logging.getLogger(__name__)
 
 
-class EmojiCount(commands.Cog):
-    """Command that give random emoji based on category."""
+class Emoji(commands.Cog):
+    """A collection of commands related to emojis in the server."""
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
     @staticmethod
-    def embed_builder(emoji: dict) -> Tuple[discord.Embed, List[str]]:
+    def embed_builder(emoji: dict) -> Tuple[Embed, List[str]]:
         """Generates an embed with the emoji names and count."""
-        embed = discord.Embed(
+        embed = Embed(
             color=Colours.orange,
             title="Emoji Count",
```

Comment:
Please end sentences in full stops.

Do this for the other cases too. Not commenting everywhere.

Reply:
Titles shouldn't end in periods, it's unconventional unless it's a fully structured sentence.

Reply:
Alright! Then it is meant for the other parts of the code. There are 2-3 cases.

Reply:
I had a look over the code but I'm unable to see more than 1 string that may require a period to be added.

If you meant for each of the list items in the `.emoji info` embed, they're not really at the level of a fully constructed sentence either. The closest may be the Created line due to humanisation, but I think it's too ambiguous to enforce and would clash with the rest of the output as it would be the only line to do so. Better to just leave it in my opinion. 

If there was any other places and I've just missed them, it may be best to specify, apologies.

Reply:
Actually on a detailed look, it appears that even that one I noticed isn't part of the diff here, instead looks to be from previous commits, so there may be more from code outside of the range I'm reading over.

Reply:
Not part of this PR (5th one is part of this PR). But will be good if it would be done as they part of the `emoji` command:

1. https://github.com/python-discord/sir-lancebot/blob/af22fe556a3b1ab8a98245cc8f96e1d81d0f4289/bot/exts/evergreen/emoji.py#L37

2. https://github.com/python-discord/sir-lancebot/blob/af22fe556a3b1ab8a98245cc8f96e1d81d0f4289/bot/exts/evergreen/emoji.py#L39

3. https://github.com/python-discord/sir-lancebot/blob/af22fe556a3b1ab8a98245cc8f96e1d81d0f4289/bot/exts/evergreen/emoji.py#L46

4. https://github.com/python-discord/sir-lancebot/blob/af22fe556a3b1ab8a98245cc8f96e1d81d0f4289/bot/exts/evergreen/emoji.py#L48

5. https://github.com/python-discord/sir-lancebot/blob/af22fe556a3b1ab8a98245cc8f96e1d81d0f4289/bot/utils/time.py#L84

Reply:
The only thing suggested here that's part of my PR is not a sentence, so I don't see a reason for the period at the end. I'll tweak the other ones.